Signing/node-express/src/views/create-order.ejs

```diff
@@ -23,19 +23,25 @@
   </div>
   <div class="mb-6">
     <label for="signatory" class="block text-gray-700 text-sm font-bold mb-2">
-      Add Signatory
+      Add Signatories
+      <p class="text-xs text-gray-500 font-normal">Optional, up to three</p>
     </label>
-    <div id="signatories-container">
-      <div class="flex items-center signatory">
-        <input class="w-full px-4 py-2 text-gray-700 border rounded focus:outline-none focus:border-blue-500" id="signatory" name="signatory" type="text" placeholder="Enter Signatory Reference" />
+    <div class="flex-col flex-wrap" id="signatories-container">
+      <div class="input-group relative mt-1 flex items-center">
+        <input class="block w-full px-4 py-2 text-gray-700 border rounded focus:outline-none focus:border-blue-500" id="signatoryOne" name="signatoryOne" type="text" placeholder="Enter Signatory Reference" />
```

Comment:
I believe you might be able to use something like `name="signatory[]"` or `name="signatory"` and have an array on the server